libnavigation-core/src/main/java/com/mapbox/navigation/core/lifecycle/MapboxNavigationApp.kt

```diff
@@ -172,16 +181,38 @@ object MapboxNavigationApp {
      * Provides access to any registered observer instance. If no observers have been registered
      * with this class type, an empty list is returned.
      */
+    @JvmStatic
     fun <T : MapboxNavigationObserver> getObservers(kClass: KClass<T>): List<T> {
         return mapboxNavigationAppDelegate.getObservers(kClass)
     }
 
+    /**
+     * Java interoperability. Provides access to any registered observer instance. If multiple
+     * instances of the same class have been registered, the first observer will be returned.
+     *
+     * @throws IllegalStateException when the class has not been registered.
```

Comment:
Don't we want to add `@Throws` annotation? It provides better java interoperability. Although in this case it doesn't matter since `IllegalStateException` is unchecked. However, I think it's a good pattern.